examples/eth_provider/config/config.exs

```diff
@@ -0,0 +1,5 @@
+import Config
+
+config :eth_provider, EthProvider,
+  client_type: Ethereumex.HttpClient,
+  node_url: System.get_env("ETH_NODE_URL")
```

Comment:
What if the variable is not set? What will happen?

Reply:
We will get an error like this:
![image](https://github.com/Subvisual-Academy/wrapeth/assets/56940846/54bd6b01-4a62-40ed-951d-c052f4f88b18)

Reply:
Exactly. Would you prefer to set a fallback value? Or to raise a specific exception?

Reply:
I have decided to raise a specific exception. I believe that it will be clearer for the user to understand what went wrong.

Reply:
Yep that's it 💪